docs/versioning.rst

```diff
@@ -0,0 +1,96 @@
+**********
+Versioning
+**********
+
+
+Current Status
+==============
+
+When a new major version is released it becomes the current release line, where all new features, bug fixes and security patches land.
```

Comment:
do we want this below the table? seems more fitting to have this statement near the definitions of current/development/etc.

do we need a definition of "release line"?

Reply:
I've opted remove this text and not refer to releases as "release lines". I think "line" is more relevant to when we discuss our git branching, which is beyond the scope of this versioning policy.